Apsitvarkom.UnitTests/Api/Controllers/CleaningEventControllerTests.cs

```diff
@@ -86,10 +86,14 @@ public async Task GetAll_RepositoryReturnsCleaningEvents_OKActionResultReturned(
         Assert.That(result.Value, Is.Not.Null.And.InstanceOf<IEnumerable<CleaningEventResponse>>());
         var resultEvents = result.Value as IEnumerable<CleaningEventResponse>;
         Assert.That(resultEvents, Is.Not.Null.And.Count.EqualTo(CleaningEvents.Count()));
+
+        IEnumerable<CleaningEvent> sortedRepositoryEvents = CleaningEvents.OrderByDescending(o => o.StartTime);
+        IEnumerable<CleaningEventResponse> sortedResultEvents = resultEvents.OrderByDescending(o => o.StartTime);
```

Comment:
You are testing the outcome of the controller method. As you've made the changes in the controller, its result should be already sorted, right?
This is why this line where you again sort the results is not necessary - you should just test the result as is.

Reply:
oh ye, am stupid. I should sort only results that came from repository. Controller method results are already sorted